resources/system-overview.md

```diff
@@ -0,0 +1,105 @@
+
+**Portal services** discussed above, are from a technical perspective bread-and-butter web frontends: serving static content, interfacing with authentication services, serving information from the datastore, mutating the datastore.
+
+**Data ingestion jobs** are scheduled to pick up batch data from sources and update the datastore with new information. This might include picking up scans of documents from a drop box and attaching them to cases, or assigning them to a task queue.
+
+**Renewal jobs** are a special form of scheduled job that pull data for individuals matching some criteria (usually those with eligibility end dates some fixed number of days in the future), evaluating that information according to plan rules, and writing determinations back to the datastore. It seems to often be the case that these jobs do not communicate to the datastore directly but expect a bundle of data to be attached to each individual before the run as part of the selection process. This selection and bundling process often seems to be fragile, with failures or inconsistencies encountered during this process causing individuals to be needlessly excluded from further automatic processing.
+
+**Notification jobs** pull data for individuals or households in a given state and generate notifications. These could be text messages, or emails, or even paper notices sent by US Mail. Renewal jobs often have the side effect of triggering these notifications directly. Sometimes they are implemented as separate jobs which facilitates policy changes that might require temporarily delaying or even canceling the delivery of a notice. The output of these jobs aften flow to vendors who run the actual mail room, SMS interfaces, or call centers.
+
+**Reaper Jobs** scan for individuals who have failed to respond to a request for information within the allowed time and disenroll them from the program. These are a common source of detritus in the backlog. During normal operations an unanswered request would result in disenrollment and case closure. Continuous eligibility during the public health emergency resulted in those cases remaining enrolled and open. The changes to implement continuous eligibility in 2020 rarely seemed to include a plan for dealing with these stale notices. Very often the existence of a pending request will exclude individuals or entire households from automatic eligibility determination, even if the request is no longer irrelevant, or only relevant to a particular individual or a non-Medicaid program. The stale requests go by many names: flags, clocks, tasks.
+
+### Data Sources
+Most sources will be required for both application and renewal. Often data sources will specify a real time API for applications to facilitate low-latency responses to user interaction on a portal, and a batch interface to allow for renewal operations to avoid overloading the real-time API.
+
+The federal Data Hub provides proxy interface to a number of data sources:
```

Comment:
This should be `Federal Data Services Hub (FDSH)`

Reply:
done!